templates/includes/sidebar.html

```diff
@@ -1,6 +1,16 @@
 <ul class="sidebar-nav">
-  <li><a href="{% url "taxi:index" %}">Home page</a></li>
-  <li><a href="{% url "taxi:manufacturer-list" %}">Manufacturers</a></li>
-  <li><a href="{% url "taxi:car-list" %}">Cars</a></li>
-  <li><a href="{% url "taxi:driver-list" %}">Drivers</a></li>
+  {% if user.is_authenticated %}
+{#    <p>User: {{ user.username }}</p>#}
+      <a href="{% url 'taxi:driver-detail' user.pk %}">User: {{ user.first_name }} {{ user.last_name }}</a>
```

Comment:
Ensure that the `driver-detail` URL pattern accepts a primary key (`pk`) as an argument. If the `user` object does not have a `pk` attribute or if the URL pattern does not expect it, this could lead to a `NoReverseMatch` error.